.abapgit.xml

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
can you reset the XML files to original for those where the data is not changed?

I'm pretty sure this is due to some endianness problems in abapGit